fbgemm_gpu/docs/source/cpp-api/merge_pooled_embeddings.rst

```diff
@@ -0,0 +1,20 @@
+Pooled Embeddings Operators
```

Comment:
I'm not sure what goes wrong, but the page builds with warnings https://deploy-preview-1165--eclectic-stroopwafel-199537.netlify.app/cpp-api/merge_pooled_embeddings.html

Reply:
https://deploy-preview-1165--eclectic-stroopwafel-199537.netlify.app/cpp-api/merge_pooled_embeddings.html fixed